CppSciComp/CppSciComp/Book.hpp

```diff
@@ -0,0 +1,29 @@
+//
+//  Book.hpp
+//  CppSciComp
+//
+//  Created by Justin Owen on 1/12/19.
+//  Copyright © 2019 Justin Owen. All rights reserved.
+//
+
+#ifndef Book_hpp
+#define Book_hpp
+
+#include <stdio.h>
+#include <string>
+
+class Book{
+public:
+    Book();
+    std::string author, title, publisher, format;
```

Comment:
For member variables, typical to follow one of two conventions:
`std::string m_author`
or
`std::string author_`

Reply:
Also, for member variables, individual declarations on newlines